src/components/HearingOutcome/HearingOutcome.tsx

```diff
@@ -0,0 +1,244 @@
+import { AnnotatedHearingOutcome } from "@moj-bichard7-developers/bichard7-next-core/build/src/types/AnnotatedHearingOutcome"
+import { Table, Tabs } from "govuk-react"
+import { useState } from "react"
+
+interface Props {
+  aho: AnnotatedHearingOutcome
+}
+
+const HearingOutcome: React.FC<Props> = ({ aho }) => {
+  const [currentTab, setCurrentTab] = useState(0)
```

Comment:
We can't really start using `useState` yet until we're sure about IE versions (despite it being likey we can).

Any chance you can just take the `<Tabs>` out of this? Not too worried if the component is messy/unwieldy at this point, it'll get redesigned as part of the upcoming details page redesign.